pkg/kubelet/kubelet.go

```diff
@@ -852,7 +852,7 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 	}
 	klet.AddPodSyncLoopHandler(activeDeadlineHandler)
 	klet.AddPodSyncHandler(activeDeadlineHandler)
-
+	klet.admitHandlers.AddPodAdmitHandler(klet.containerManager.GetTopologyPodAdmitHandler())
```

Comment:
I would probably add this line as part of https://github.com/kubernetes/kubernetes/pull/74357 so that this current PR can be merged independent of all other PRs in this chain.